Makefile

```diff
@@ -100,6 +99,14 @@ lint: golangci-lint ## Run the golangci-lint linter.
 lint-fix: golangci-lint ## Run the golangci-lint linter and perform fixes.
 	$(GOLANGCI_LINT) run --fix
 
+.PHONY: tidy
+tidy:
+	go mod tidy
+
+.PHONY: vendor
+vendor: tidy
+	go mod vendor
+
```

Comment:
Looks like you patched Makefile (a one-time manual step), and then 'go mod vendor' becomes the command to automate the rest of the patch (a step that can be repeated in the future when we bump to newer dependencies).  Splitting that into two patches may make it easier to distinguish the manual changes from the automated ones.

Reply:
I would not separate these, I used `make vendor` to generate this entire PR